packages/react-draggable-dialog/src/components/DraggableDialogSurface/useDraggableDialogSurface.ts

```diff
@@ -14,7 +14,8 @@ import {
  * Returns the state needed to make a draggable dialog surface.
  */
 export const useDraggableDialogSurface = (
-  props: DraggableDialogSurfaceProps
+  props: DraggableDialogSurfaceProps,
+  _ref: React.ForwardedRef<HTMLDivElement>
```

Comment:
dtto

Reply:
Here as well, the name `ref` is already in use. (Line 39)